### PR TITLE
Readme code improvement

### PR DESCRIPTION
**Pull request by vrkansagara**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,26 +3,27 @@
...
-php -S localhost:8001 -t public 
+cd path/to/clone/directory
+php -S 0.0.0.0:8001 -t public public/index.php
 ```
 
-Open [localhost:8001](http://localhost:8001) in your browser to see the website
+Open [0.0.0.0:8001](http://0.0.0.0:8001) in your browser to see the website
 
-## Create Static Website to Deploy
+## Create static website to deploy
```
Comment (TomasVotruba) [inline README.md:25]: English Headlines must be capical

Comment (vrkansagara) [reply README.md:25]: @TomasVotruba  you are right but I found my self for the english  sentence formation like **The First Letter in a Sentence should be captial**

Comment (froschdesign) [reply README.md:25]: There are different styles and the current one looks like the Chicago Manual of Style.

Here is a tool that helps with capitalization in titles and which describes the different styles: https://capitalizemytitle.com

Comment (vrkansagara) [reply README.md:25]: Lets add style code into contribution guide line.

**Comment by TomasVotruba** [comment]
Closing as feedback was not resolved yet.

Feel free to re-open with content-change PR only

**Comment by vrkansagara** [comment]
> Closing as feedback was not resolved yet.
> 
> Feel free to re-open with content-change PR only
:+1:

**Pull request closed without merging**